src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisCatchVariables.java

```diff
@@ -46,4 +46,16 @@ public void run() {
 
     private static void exceptional(Throwable ex) {}
     private static void debug(String message, Throwable err) {}
+
+    int baseline;
+    public int getBaseline() {
+        int baseline = 12;
+        final int placement = 3;
+        switch (placement) {
+            case 3:
+                baseline = 3 + baseline;
+                return baseline;
+        }
+        return -1;
+    }
```

Comment:
There is no harm in keeping these test cases, they were killing some mutations when sets were replaced with lists. Same for the other example.